dashboard/src/components/Common/Autocomplete/AutocompleteField.scss

```diff
@@ -20,14 +20,27 @@
 
 .suggestion {
   border-bottom: 1px solid $input-border;
-  padding: 12px;
 
   &:last-child {
     border: none;
   }
+
+  &:first-child {
+    @extend .suggestionFocused;
```

Comment:
On my computer, this keeps the first child selected at all times:

<img width="511" alt="screen shot 2016-11-19 at 10 29 16 am" src="https://cloud.githubusercontent.com/assets/464857/20457577/21b1bc46-ae43-11e6-9ae6-5c594471cd01.png">

Do you think there might be a way to do this so that the first child is selected *logically*, not just in CSS?